lib/rubocop/cop/style/collection_compact.rb

```diff
@@ -85,6 +87,7 @@ class CollectionCompact < Base
         PATTERN
 
         def on_send(node)
+          return if target_ruby_version < 2.6 && %i[filter filter!].include?(node.method_name)
```

Comment:
Can you define `%i[filter filter!]` as a constant? (e.g., `FILTER_METHODS`)

```suggestion
          return if target_ruby_version < 2.6 && FILTER_METHODS.include?(node.method_name)
```